### ISSUE TITLE
Adguard "Non-filtering" DNS as Default

### ISSUE THREAD

**Issue by ZExplore** [AdguardTeam/AdGuardHome#1517]
> @ameshkov Well, we can consider using AdGuard DNS by default in one ane only case -- if we add an endpoint that does no ad & tracking filtering. But it kinda defeats the purpose, the idea of AdGuard DNS is to block this stuff:)

**Comment by ameshkov** [comment]
Well, yeah, we can consider it now indeed :)

**Comment by ZExplore** [comment]
v0.102?

**Comment by ArchiveBase** [comment]
> v0.102?

Done.

**Comment by ameshkov** [comment]
Nah, v1.0. Hopefully, by that time AGH will cover all necessary locations, currently, there are some blank spots.
